specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-06-01-preview/mfe.json

```diff
@@ -11405,6 +11506,16 @@
       "x-ms-discriminator-value": "AMLToken",
       "additionalProperties": false
     },
+    "AmlTokenComputeIdentity": {
```

Comment:
See all the newly introduced definitions which are missing Title/Description, in the Linter Warnings: [https://github.com/Azure/azure-rest-api-specs/pull/24051/checks?check_run_id=13666592472](https://github.com/Azure/azure-rest-api-specs/pull/24051/checks?check_run_id=13666592472).

Schema should have a description or title.

Reply:
Fixed